crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -373,6 +378,7 @@ impl Serialize for Entity {
     }
 }
 
+#[cfg(feature = "serde")]
 impl<'de> Deserialize<'de> for Entity {
```

Comment:
Same here, this way we will not have feature importing

Reply:
They're still required for reflecting the traits.